pages/windows/tracert.md

```diff
@@ -0,0 +1,28 @@
+# tracert
+
+> Receive information about each step in the route between your PC and the target.
+> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb491018(v=technet.10)?redirectedfrom=MSDN>.
+
+- Trace a route:
+
+`tracert {{IP}}`
+
+- Prevent tracert from resolving IP addresses to hostnames:
+
+`tracert -d {{IP}}`
+
+- Force tracert to use IPv4 only:
+
+`tracert -4 {{IP}}`
+
+- Force tracert to use IPv6 only:
+
+`tracert -6 {{IP}}`
+
+- Show detailed help:
+
+`tracert /?`
+
+- Specify the maximum number of hops in the search for the target:
+
+`tracert -h {{max_hops}} {{IP}}`
```

Comment:
```suggestion
- Specify the maximum number of hops in the search for the target:

`tracert /h {{max_hops}} {{IP}}`

- Display help:

`tracert /?`
```

`Display help` is the generic description we use in most of our pages, also we list it at last.